composer.json

```diff
@@ -12,8 +12,13 @@
             "dealerdirect/phpcodesniffer-composer-installer": true
         }
     },
+    "conflict": {
+        "amphp/dns": "<2.4.0",
+        "phpunit/phpunit": "<9.6.22",
+        "amphp/socket": "<2.3.1"
+    },
     "require": {
-        "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0",
+        "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
```

Comment:
I think php 8.0 can be dropped, then update platform -> php config in this composer.json to 8.1.99